README.md

```diff
@@ -4,7 +4,7 @@
 ![CI](https://github.com/yannikschaelte/yaml2sbml/workflows/CI/badge.svg)
 [![codecov](https://codecov.io/gh/yannikschaelte/yaml2sbml/branch/master/graph/badge.svg?token=AORT3GREQJ)](https://codecov.io/gh/yannikschaelte/yaml2sbml)
 
-<img src="https://raw.githubusercontent.com/yaml2sbml-dev/yaml2sbml/feature_logo/doc/logo/logo_yaml2sbml_long.png" width="80%" alt="yaml2sbml logo"/>
+![yaml2sbml logo](doc/logo/logo_yaml2sbml_long.png)
```

Comment:
The problem is that we copy the README.md onto pypi as well, and there the relative path will break. Thus I think we need the absolute URL.

Reply:
Fully agreed though that this was the wrong link. I think one should replace feature_logo by master and then quickly merge things there. Also needs to be done in doc/index.rst in #81

Reply:
```suggestion
![yaml2sbml logo](https://github.com/yaml2sbml-dev/yaml2sbml/blob/master/doc/logo/logo_yaml2sbml_long.png?raw=true)
```